label_studio_converter/converter.py

```diff
@@ -314,21 +322,40 @@ def convert_to_coco(self, input_data, output_dir, output_image_dir=None, is_dir=
                         'name': category_name
                     })
                 category_id = category_name_to_id[category_name]
+
                 annotation_id = len(annotations)
-                x = int(bbox['x'] / 100 * width)
-                y = int(bbox['y'] / 100 * height)
-                w = int(bbox['width'] / 100 * width)
-                h = int(bbox['height'] / 100 * height)
-                annotations.append({
-                    'id': annotation_id,
-                    'image_id': image_id,
-                    'category_id': category_id,
-                    'segmentation': [],
-                    'bbox': [x, y, w, h],
-                    'ignore': 0,
-                    'iscrowd': 0,
-                    'area': w * h
-                })
+
+                if "rectanglelabels" in label:
+                    x = int(label['x'] / 100 * width)
+                    y = int(label['y'] / 100 * height)
+                    w = int(label['width'] / 100 * width)
+                    h = int(label['height'] / 100 * height)
+
+                    annotations.append({
+                        'id': annotation_id,
+                        'image_id': image_id,
+                        'category_id': category_id,
+                        'segmentation': [],
+                        'bbox': [x, y, w, h],
+                        'ignore': 0,
+                        'iscrowd': 0,
+                        'area': w * h
+                    })
+                elif "polygonlabels" in label:
+                    x, y = zip(*label["points"])
+
+                    annotations.append({
+                        'id': annotation_id,
+                        'image_id': image_id,
+                        'category_id': category_id,
+                        'segmentation': [sum(label["points"], [])],
```

Comment:
Segmentation & bbox coordinates are coming in % , so they have to be converted to image absolute coordinates (like wise it's done in `"rectanglelabels"` case)